flyteadmin/pkg/rpc/adminservice/project.go

```diff
@@ -63,3 +63,21 @@ func (m *AdminService) UpdateProject(ctx context.Context, request *admin.Project
 
 	return response, nil
 }
+
+func (m *AdminService) GetProject(ctx context.Context, request *admin.Project) (*admin.Project, error) {
+	defer m.interceptPanic(ctx, request)
+	if request == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Incorrect request, nil requests not allowed")
+	}
+	var response *admin.Project
+	var err error
+	m.Metrics.projectEndpointMetrics.get.Time(func() {
+		response, err = m.ProjectManager.GetProject(ctx, *request)
+	})
+	if err != nil {
+		return nil, util.TransformAndRecordError(err, &m.Metrics.projectEndpointMetrics.list)
```

Comment:
```suggestion
		return nil, util.TransformAndRecordError(err, &m.Metrics.projectEndpointMetrics.get)
```

Reply:
this is still re-using the list metrics, mind adding a separate counter for get?